OwinOAuthProvidersDemo/App_Start/Startup.Auth.cs

```diff
@@ -20,293 +20,300 @@ public void ConfigureAuth(IAppBuilder app)
 			});
 			// Use a cookie to temporarily store information about a user logging in with a third party login provider
 			app.UseExternalSignInCookie(DefaultAuthenticationTypes.ExternalCookie);
-            //app.UseDeviantArtAuthentication("id", "secret");
-            //app.UseUntappdAuthentication("id", "secret");
-            // Uncomment the following lines to enable logging in with third party login providers
-            //app.UseMicrosoftAccountAuthentication(
-            //    clientId: "",
-            //    clientSecret: "");
+			//app.UseDeviantArtAuthentication("id", "secret");
```

Comment:
Whats the change here?

Reply:
I use tabs instead of spaces in my VS settings. I've changed the entire file back to spaces.